packages/@atjson/source-gdocs-paste/src/converter.ts

```diff
@@ -183,6 +201,34 @@ GDocsSource.defineConverterTo(OffsetSource, (doc) => {
       }
     });
 
+  doc.where({ type: "-gdocs-ps_al" }).update((align) => {
+    let alignment: "start" | "center" | "end" | "justify" | undefined;
```

Comment:
Would an alignment of undefined cause issues in the renderer?

Reply:
Nope! Undefined alignment is ok :)